test/hooks/test-trace-express.js

```diff
@@ -123,6 +126,23 @@ describe('test-trace-express', function() {
       common.doRequest('GET', done, expressPredicate);
     });
   });
+
+  it('should have proper labels', function(done) {
+    var app = express();
+    app.get('/', function (req, res) {
+      res.send(common.serverRes);
+    });
+    server = app.listen(common.serverPort, function() {
+      http.get({port: common.serverPort}, function(res) {
+        var labels = common.getMatchingSpan(expressPredicate).labels;
+        assert.equal(labels[traceLabels.HTTP_RESPONSE_CODE_LABEL_KEY], '200');
+        assert.equal(labels[traceLabels.HTTP_METHOD_LABEL_KEY], 'GET');
+        assert.equal(labels[traceLabels.HTTP_URL_LABEL_KEY], 'http://localhost/');
+        assert.equal(labels[traceLabels.HTTP_SOURCE_IP], '::ffff:127.0.0.1');
```

Comment:
This is dependent on the machine network stack. Maybe check for presence of the property rather than the value.